cmd/shove/main.go

```diff
@@ -150,9 +152,31 @@ func main() {
 			slog.Error("Failed to add WebPush service", "error", err)
 			os.Exit(1)
 		}
-	}
-
-	if *telegramBotToken != "" {
+	} else if *webPushVAPIDKeysJSON != "" {
+		content, err := os.ReadFile(*webPushVAPIDKeysJSON)
```

Comment:
Would be good to add a check here that both `webPushVAPIDPublicKey` and `webPushVAPIDPrivateKey` are empty.